src/main/scala/uk/gov/nationalarchives/dp/client/Entities.scala

```diff
@@ -10,28 +10,29 @@ object Entities {
       entityType: Option[String],
       ref: UUID,
       title: Option[String],
+      description: String,
       deleted: Boolean,
       path: Option[String]
   )
 
-  def fromType[F[_]](entityType: String, ref: UUID, title: Option[String], deleted: Boolean)(implicit
-      me: MonadError[F, Throwable]
+  def fromType[F[_]](entityType: String, ref: UUID, title: Option[String], description: String, deleted: Boolean)(
```

Comment:
Would this be better off as an `Option[String]` if it's not present on all of the entities? It might get a bit confusing if we can't tell the difference between a missing description and a blank one.